packages/typespec-vscode/src/utils.ts

```diff
@@ -30,6 +31,17 @@ export async function isDirectory(path: string) {
   }
 }
 
+export async function createTempDir(): Promise<string | undefined> {
+  try {
+    const tempDir = tmpdir();
+    const realTempDir = await realpath(tempDir);
+    return await mkdtemp(realTempDir);
```

Comment:
I think you don't need to make the temp for above, the arg is just a prefix to find it in the global user temp dir

Reply:
Thanks. The correct usage is `await mkdtemp(join(tmpdir(), 'foo-'));`, per API docs: https://docs.deno.com/api/node/fs/promises/~/mkdtemp

`mkdtemp()` will append 6 random characters to the given prefix and create it.